sorbet/rbi/shims/gems/yarp.rbi

```diff
@@ -0,0 +1,216 @@
+# typed: strict
```

Comment:
You might want to use the RBI @vinistock added in https://github.com/Shopify/ruby-lsp/pull/978

Reply:
Without more automation this is going to go out of date pretty quickly